AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -1749,7 +1749,6 @@
 				058D0A39195D057000B7D73C /* ASDisplayNodeAppearanceTests.m in Sources */,
 				CCB2F34D1D63CCC6004E6DE9 /* ASDisplayNodeSnapshotTests.m in Sources */,
 				AE6987C11DD04E1000B9E458 /* ASPagerNodeTests.m in Sources */,
-				699B83511E3C1BA500433FA4 /* ASLayoutSpecTests.m in Sources */,
```

Comment:
This seems to come up every time I open the project in Xcode. Unless someone objects, let's let this ride-along with this diff.